glide.yaml

```diff
@@ -57,8 +57,11 @@ import:
   - package: github.com/evergreen-ci/pail
   - package: gonum.org/v1/gonum
   - package: github.com/evergreen-ci/timber
-
   - package: github.com/evergreen-ci/certdepot
+  - package: github.com/shirou/gopsutil
+  - package: github.com/StackExchange/wmi
+  - package: github.com/go-ole/go-ole
+  - package: github.com/shirou/w32
```

Comment:
Is there a reason we need these other packages? Are they dependencies of gopsutil?

Reply:
For some reason the Windows tests threw an error because these dependencies were missing so I added them

Reply:
Can you add a comment above gopsutil saying `# gopsutil and its dependencies`?

Reply:
done